numberscope/frontscope
# Add Vue Test Utils

This commit adds Vue Test Utils, which is "a set of utility
functions aimed to simplify testing Vue.js components". It
allows us to unit test Vue components.

I was originally trying to set up the Jest testing framework for general testing purposes, but I was running into lots of issues. My plan is to further configure Jest for general testing purposes now that I've got Vue Test Utils working.

To test this PR:

1. `npm run test:unit` should output something like the following
```
> newnumberscope@0.2.0 test:unit
> vue-cli-service test:unit

 PASS  tests/unit/SeqGetter.spec.ts
  ✓ displays dummy title (21ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        3.828s
Ran all test suites.
```
2. `npm run serve` should compile successfully
3. you should be able to interact with Numberscope in your browser as usual
